src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -215,37 +215,38 @@ void GeantPhysicsList::add_gamma_processes()
 /*!
  * Add EM processes for electrons and positrons.
  *
- * | Processes                    | Model classes             |
- * | ---------------------------- | --------------------------|
- * | Pair annihilation            | G4eeToTwoGammaModel       |
- * | Ionization                   | G4MollerBhabhaModel       |
- * | Bremsstrahlung (low E)       | G4SeltzerBergerModel      |
- * | Bremsstrahlung (high E)      | G4eBremsstrahlungRelModel |
- * | Coulomb scattering           | G4eCoulombScatteringModel |
- * | Multiple scattering (low E)  | G4UrbanMscModel           |
- * | Multiple scattering (high E) | G4WentzelVIModel          |
+ * | Processes                    | Model classes                |
+ * | ---------------------------- | ---------------------------- |
+ * | Pair annihilation            | G4eeToTwoGammaModel          |
+ * | Ionization                   | G4MollerBhabhaModel          |
+ * | Bremsstrahlung (low E)       | G4SeltzerBergerModel         |
+ * | Bremsstrahlung (high E)      | G4eBremsstrahlungRelModel    |
+ * | Coulomb scattering           | G4eCoulombScatteringModel    |
+ * | Multiple scattering (low E)  | G4UrbanMscModel              |
+ * | Multiple scattering (low E)  | G4GoudsmitSaundersonMscModel |
+ * | Multiple scattering (high E) | G4WentzelVIModel             |
  *
  * \note
- * - Bremsstrahlung models are selected manually at compile time using
- *   \c GeantBremsstrahlungProcess::ModelSelection and need to be updated
- *   accordingly.
  * - Coulomb scattering and multiple scattering (high E) are currently
  *   disabled.
  */
 void GeantPhysicsList::add_e_processes(G4ParticleDefinition* p)
 {
     auto* physics_list = G4PhysicsListHelper::GetPhysicsListHelper();
 
-    if (p == G4Positron::Positron())
+    if (options_.annihilation)
```

Comment:
```suggestion
    if (options_.annihilation && p == G4Positron::Positron())
```
and remove the extra braces.